lib/math.rb

```diff
@@ -0,0 +1,29 @@
+module Math
+  def average(array, round_precision)
+    (sum(array) / array.length).round(round_precision)
+  end
+
+  def average_by(total_sum, total_length, round_precision)
```

Comment:
A thought on this method.

Should it also take an array instead of the sum?  That means the caller doesn't have to do its own summing, and instead just relies on the math module to do that math?